test/presenters/content_item/contents_list_test.rb

```diff
@@ -78,4 +78,45 @@ def body
       { text: "Four", id: "four" }
     ], @contents_list.contents_items
   end
+
+  test "#show_contents_list? returns true if number of contents items is less than 3 and the first item's character count is above 100" do
+    class << @contents_list
+      def body
+        "<h2 id='one'>One</h2>
```

Comment:
I'm not sure how we can avoid this, but it feels a little fragile. If somebody updates the heading structure and all the items in govspeak become h3s then this won't happen on the live site, but the tests will continue to pass.

Reply:
The [existing implementation relies on the presence of `h2`s](https://github.com/alphagov/government-frontend/blob/master/app/presenters/content_item/contents_list.rb#L19), it's not a problem we're solving here. It's also a pattern we use in Govspeak frequently for anchored headings.